velox/common/memory/MemoryAllocator.h

```diff
@@ -164,23 +164,12 @@ class MemoryAllocator : public std::enable_shared_from_this<MemoryAllocator> {
   enum class Kind {
     /// The default memory allocator kind which is implemented by
     /// MemoryAllocatorImpl. It delegates the memory allocations to std::malloc.
-    kStd,
+    kMalloc,
     /// The memory allocator kind which is implemented by MmapAllocator. It
     /// manages the large chunk of memory allocations on its own by leveraging
     /// mmap and madvice, to optimize the memory fragmentation in the long
     /// running service such as Prestissimo.
     kMmap,
-    /// The memory allocator kind which is implemented by AsyncDataCache to
-    /// integrate with the file cache management. It is a wrapper on top of an
-    /// actual memory allocator. It delegates the memory allocation to the
-    /// associated memory allocator and might evict file cache to make space for
-    /// the memory allocations and retries on allocation failures.
-    kCache,
```

Comment:
What's the motivation for this change? Would you update PR description to clarify?

Reply:
I put it PR description. Thanks!